### PR TITLE
Administration/onboarding checklist

### PR DESCRIPTION
**Pull request by jsooter**
Onboarding Checklist to assist with setting up Postman teams. This doc has been reviewed by the onboarding-squad and also the security team.

### CODE REVIEW

File: src/pages/docs/administration/onboarding-checklist.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+* Your IT team may need to add an exception to device policy allowing for Postman to be installed on employee’s workstations. Provide a [Postman download link​](https://www.getpostman.com/downloads/) to the IT team to help establish this exception.
+* If your organization's network connection is facilitated via a proxy, you may need to configure Postman appropriately. Retrieve proxy connection details from your IT team and [​setup within Postman](https://learning.getpostman.com/docs/postman/sending_api_requests/proxy/)​.
+* It is recommended that your IT team whitelist Postman’s domains to ensure Postman data is synced with the cloud and all functionality works as expected. By default, WebSocket connections use the same ports as HTTP (80) and HTTPS (443). Ensure your network firewall allows WebSocket connections for Postman's services by whitelisting the following domains:
+
+    * *.postman.com
```
Comment (SueSmith) [inline src/pages/docs/administration/onboarding-checklist.md:43]: I'd maybe stick these in backticks i.e. `*.postman.com` etc

File: src/pages/docs/administration/onboarding-checklist.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    * *.getpostman.com
+    * *.postman.co
+    * *.pstmn.io
+
+* Depending on your [plan](https://www.postman.com/pricing) static IP addresses may be available for Postman Monitors. Your IT team should whitelist these static IPs for monitoring. Currently, static IPs are available only for the US east and US west regions. If you selected a Static IP region on your monitors, you will have to whitelist the following IP addresses:
+
+    * 34.201.186.27 for US - east
```
Comment (SueSmith) [inline src/pages/docs/administration/onboarding-checklist.md:50]: same again re backticks around the ip addresses

File: src/pages/docs/administration/onboarding-checklist.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    * 52.89.173.88 for US - west
+
+* Update your [team settings](https://go.postman.co/settings/team/general). You can find out more about the team settings in [Team Settings](/docs/postman/collaboration/team-settings/) on the Learning Center.
```
Comment (SueSmith) [inline src/pages/docs/administration/onboarding-checklist.md:53]: second sentence I'd just condense a wee bit e.g. "See [Team Settings](/docs/postman/collaboration/team-settings/) for more info."

File: src/pages/docs/administration/onboarding-checklist.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+* If your Postman [plan](https://www.postman.com/pricing) includes single-sign-on (SSO) you may need help from your IT team to configure it. For step by step instructions for integrating with a number of identity providers please see [Configuring SSO for a team](/docs/administration/sso/admin-sso/) on the Learning Center.
```
Comment (SueSmith) [inline src/pages/docs/administration/onboarding-checklist.md:54]: similar for LC link here e.g. "See [Configuring SSO for a team](/docs/administration/sso/admin-sso/) for instructions on integrating with specific identity providers."

File: src/pages/docs/administration/onboarding-checklist.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+### Invite Team Members
+
+Depending on the size of you team you can send invites to your team now or [Check Your Setup](#check-your-setup) with a few volunteers before you rollout Postman to your entre organization.
```
Comment (SueSmith) [inline src/pages/docs/administration/onboarding-checklist.md:58]: wee typo "you team" > "your team"

File: src/pages/docs/administration/onboarding-checklist.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+
+Either way, you can add your teammates from your [team dashboard](https://go.postman.co/team) or by generating a shareable link or inviting them through a direct email. You can find out how to do that in [Managing Your Team](/docs/postman/collaboration/managing-your-team/#inviting-to-a-team) on the Learning Center.
```
Comment (SueSmith) [inline src/pages/docs/administration/onboarding-checklist.md:60]: I think I'd lose the "either way" and add a comma e.g. "You can add your teammates from your [team dashboard](https://go.postman.co/team), by generating a shareable link, or inviting them through a direct email."

Then something like "See [Managing your team](/docs/postman/collaboration/managing-your-team/#inviting-to-a-team) for more detail."

File: src/pages/docs/administration/onboarding-checklist.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+
+![Invite Users](https://assets.postman.com/postman-docs/Screen%20Shot%202019-11-12%20at%201.18.07%20PM.png)
+
+### Check Your Setup
+
+Before we kick-off in full, it’s best for us to check-in on the workings of the Postman app within your organization and ensure everything works as expected. If you do not have the Postman app and Postman account find a team member who does to help you test the configuration. Depending on your organization's IT policy you may need to make requests of your IT team in order to get up and running as well.
```
Comment (SueSmith) [inline src/pages/docs/administration/onboarding-checklist.md:66]: we tend not to use first person so maybe reword to "Before you kick-off in full, it's best to check..."

File: src/pages/docs/administration/onboarding-checklist.md [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+
+Ensure you are connected to your organization's network and test the following:
+
+* Can you ​[execute a request](/docs/postman/sending_api_requests/requests)​ to example.com within Postman?
```
Comment (SueSmith) [inline src/pages/docs/administration/onboarding-checklist.md:70]: maybe stick example.com in backticks

File: src/pages/docs/administration/onboarding-checklist.md [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+* Can you execute a request to one of your internal/private APIs?
+* Save a request inside a [​Collection​](/docs/postman/collections/creating_collections). Visit ​go.postman.co​, has your Collection and Request synced with Postman’s cloud?
```
Comment (SueSmith) [inline src/pages/docs/administration/onboarding-checklist.md:72]: similar for address, just makes it a bit more readable in the backticks

Comment (SueSmith) [reply src/pages/docs/administration/onboarding-checklist.md:72]: I'd remove the capitalization on Collection and Request too

File: src/pages/docs/administration/onboarding-checklist.md [hunk 10]
```diff
@@ -0,0 +1,92 @@
...
+
+Repeat these checks on a colleague’s workstation. If everything worked, your workstations and network are ready to support Postman! Else, if you experienced a hiccup, [​let us know](https://getpostman.com/support) and please read on.
```
Comment (SueSmith) [inline src/pages/docs/administration/onboarding-checklist.md:74]: if we can lose the first person again e.g. "contact Postman support" instead of "let us know"

File: src/pages/docs/administration/onboarding-checklist.md [hunk 11]
```diff
@@ -0,0 +1,92 @@
...
+
+### Enable Team Discovery
+
+When team discovery is enabled, users in your organization can see teams to join when they access their Postman accounts. Anyone signing in with a company email address is presented with available teams and can make a request to join each one. Administrators will receive a notification and can approve or deny access from there.
+
+![Team Discovery](https://blog.postman.com/wp-content/uploads/2019/10/enable-discovery-768x189.png)
```
Comment (SueSmith) [inline src/pages/docs/administration/onboarding-checklist.md:80]: image is returning a 404 atm, maybe from recent blog site changes..

File: src/pages/docs/administration/onboarding-checklist.md [hunk 12]
```diff
@@ -0,0 +1,92 @@
...
+
+This kind of automation makes Postman’s collaboration features more discoverable to users, and allows your team participation to scale up along with your successful API projects.
+
+### Questions?
+
+If you have any questions or run into any issues setting up Postman for your team be sure and check to see if the question has already been answered by the [Postman Community](https://community.postman.com/). You can also find a number of support resources on our [support page](https://www.postman.com/support).
```
Comment (SueSmith) [inline src/pages/docs/administration/onboarding-checklist.md:86]: if we can lose "our" here i.e. "the support page"

File: src/pages/docs/administration/onboarding-checklist.md [hunk 13]
```diff
@@ -0,0 +1,92 @@
...
+
+## Next Steps
+
+Checkout [Security and Compliance: A Shared Responsibility Model](https://www.postman.com/shared-responsibility/) for important security considerations.
```
Comment (SueSmith) [inline src/pages/docs/administration/onboarding-checklist.md:90]: "checkout" > "check out"

File: src/pages/docs/administration/onboarding-checklist.md [hunk 14]
```diff
@@ -0,0 +1,92 @@
...
+
+Finally, head over to [Getting Started With Postman](https://learning.postman.com/getting-started/) for what to do next.
```
Comment (SueSmith) [inline src/pages/docs/administration/onboarding-checklist.md:92]: this page is actually getting removed soon, tbh it's not a great place to get started 🙈 maybe just send them to the docs homepage?

File: src/pages/docs/administration/onboarding-checklist.md [hunk 15]
```diff
@@ -0,0 +1,92 @@
...
+page_id: "onboarding_checklist"
+warning: false
+contextual_links:
+  - type: section
+    name: "Additional Resources"
+  - type: link
+    name: "Security and Compliance: A Shared Responsibility Model"
+    url: "https://www.postman.com/shared-responsibility/"
+  - type: link
+    name: "Managing your team"
```
[the remark this answers is not in the document]
Comment (jsooter) [reply src/pages/docs/administration/onboarding-checklist.md:13]: It's more of a checklist and less of a flow so I did not include any of the items in the lest as "next steps". We can try it out since Getting Started is being removed.

File: src/pages/docs/administration/onboarding-checklist.md [hunk 16]
```diff
@@ -0,0 +1,92 @@
...
+    url: "/docs/administration/sso/admin-sso/"
+  - type: subtitle
+    name: "Related Blog Posts"
+  - type: link
+    name: "Manage Large Teams in Postman with Workspaces, Permissions, and Version Control"
+    url: "https://blog.postman.com/2019/07/23/postman-team-workspaces-and-permissions/"
+
+---
+
+What is this page about?
```
[the remark this answers is not in the document]
Comment (jsooter) [reply src/pages/docs/administration/onboarding-checklist.md:26]: Rookie mistake 😊

File: src/pages/docs/administration/onboarding-checklist.md (hunk 14 above)
Comment (jsooter) [reply src/pages/docs/administration/onboarding-checklist.md]: Changed to [Managing Your Team](/docs/postman/collaboration/managing-your-team/)

File: src/pages/docs/administration/onboarding-checklist.md (hunk 13 above)
Comment (jsooter) [reply src/pages/docs/administration/onboarding-checklist.md]: done

File: src/pages/docs/administration/onboarding-checklist.md (hunk 12 above)
Comment (jsooter) [reply src/pages/docs/administration/onboarding-checklist.md]: done

File: src/pages/docs/administration/onboarding-checklist.md (hunk 11 above)
Comment (jsooter) [reply src/pages/docs/administration/onboarding-checklist.md]: removed image and revised paragraph.

File: src/pages/docs/administration/onboarding-checklist.md (hunk 10 above)
Comment (jsooter) [reply src/pages/docs/administration/onboarding-checklist.md]: done

File: src/pages/docs/administration/onboarding-checklist.md [hunk 17]
```diff
@@ -0,0 +1,92 @@
...
+* [Setting up Postman](#setting-up-postman)
+    * [Functionality Tests](#functionality-tests)
+    * [Send Invites](#send-invites)
+    * [Enable Team Discovery](#enable-team-discovery)
+
+## Setting up Postman
+
+Make sure everyone who uses Postman has the latest version by downloading it from [the Postman website](https://www.getpostman.com/downloads/). You may want to contact your IT team to establish what the procedure is for adding a new piece of software. This varies from organization to organization, however, the following points are common:
+
+* Your IT team may need to add an exception to device policy allowing for Postman to be installed on employee’s workstations. Provide a [Postman download link​](https://www.getpostman.com/downloads/) to the IT team to help establish this exception.
```
[the remark this answers is not in the document]
Comment (jsooter) [reply src/pages/docs/administration/onboarding-checklist.md]: done

File: src/pages/docs/administration/onboarding-checklist.md (hunk 1 above)
Comment (jsooter) [reply src/pages/docs/administration/onboarding-checklist.md]: done

File: src/pages/docs/administration/onboarding-checklist.md [hunk 18]
```diff
@@ -0,0 +1,92 @@
...
+    url: "https://blog.postman.com/2019/07/23/postman-team-workspaces-and-permissions/"
+
+---
+
+What is this page about?
+
+## Contents
+
+* [Setting up Postman](#setting-up-postman)
+    * [Functionality Tests](#functionality-tests)
```
[the remark this answers is not in the document]
Comment (jsooter) [reply src/pages/docs/administration/onboarding-checklist.md]: done

File: src/pages/docs/administration/onboarding-checklist.md (hunk 2 above)
Comment (jsooter) [reply src/pages/docs/administration/onboarding-checklist.md]: done

File: src/pages/docs/administration/onboarding-checklist.md (hunk 3 above)
Comment (jsooter) [reply src/pages/docs/administration/onboarding-checklist.md]: done

File: src/pages/docs/administration/onboarding-checklist.md (hunk 4 above)
Comment (jsooter) [reply src/pages/docs/administration/onboarding-checklist.md]: done

File: src/pages/docs/administration/onboarding-checklist.md (hunk 5 above)
Comment (jsooter) [reply src/pages/docs/administration/onboarding-checklist.md]: done

File: src/pages/docs/administration/onboarding-checklist.md (hunk 6 above)
Comment (jsooter) [reply src/pages/docs/administration/onboarding-checklist.md]: done

File: src/pages/docs/administration/onboarding-checklist.md (hunk 7 above)
Comment (jsooter) [reply src/pages/docs/administration/onboarding-checklist.md]: done

File: src/pages/docs/administration/onboarding-checklist.md (hunk 8 above)
Comment (jsooter) [reply src/pages/docs/administration/onboarding-checklist.md]: done

File: src/pages/docs/administration/onboarding-checklist.md (hunk 9 above)
Comment (jsooter) [reply src/pages/docs/administration/onboarding-checklist.md]: done
